### DISCUSSION TITLE
Apply `nospell` in markdown through `autocommands`

### DISCUSSION THREAD

**Discussion by jopemachine** [LunarVim/LunarVim#2629] [Q&A] [answered]
I found a similar-looking issue in https://github.com/LunarVim/LunarVim/issues/936.
So I added the below code to `config.lua`.

```lua
lvim.autocommands.custom_groups = {
    { "FileType", "markdown", "setlocal nospell" },
}
```

But `nospell` does not apply occasionally.
It looks like some kind of race condition.
How can I make this work?

**Comment by LostInTheLogs** [comment, marked as answer]
we had an autocommand that would set spell in md files, it's removed now
